CHANGELOG.md

```diff
@@ -2,10 +2,18 @@
 
 ## Unreleased
 
+## v0.131.0
+
 ### 🛑 Breaking changes 🛑
 
+- (Splunk) `discovery`: Move `splunk.continuousDiscovery` feature flag to stable
```

Comment:
we are still missing the big remove notice from https://github.com/signalfx/splunk-otel-collector/pull/6510

Reply:
I don't see the block from https://github.com/signalfx/splunk-otel-collector/pull/6510/files#r2248722952

Reply:
nvm... 🤦🏼